app/helpers/devise_helper.rb

```diff
@@ -0,0 +1,14 @@
+module DeviseHelper
+  def devise_error_messages!
+    return '' if resource.errors.empty?
+
+    messages = resource.errors.full_messages.map { |msg| content_tag(:div, msg) }.join
+    html = <<-HTML
+    <div class="alert alert-danger">
+      #{messages}
+    </div>
+    HTML
```

Comment:
Just want to confirm that you mean to have only the list of errors and not a header that says something like "2 errors prevented this xxx from saving". The header is probably unnecessary in terms of usability, so I'm fine with the way things are.

Reply:
Yes, I did mean it that way. As long as you are okay with it. 😄

Reply:
✔️

Reply:
✔️